insights/specs/default.py

```diff
@@ -89,6 +89,7 @@ def inner(idx=None):
 
 
 class DefaultSpecs(Specs):
+    abrt_status_bare = simple_command("/usr/bin/abrt status --bare=True")
```

Comment:
Since this is a `simple_command` and will be stored in the insights archive as a `simple_file` you also need to add a new simple_file spec to `insights/specs/insights_archive.py`.  The archive file spec can be determined by running the `mangle` command on your command specs like this `$ mangle "/usr/bin/abrt status --bare=True"`.  `mangle` is included in the insights-core project and should be available in your development environment.